interfaces/systemd/backend.go

```diff
@@ -78,18 +77,16 @@ func (b *Backend) Setup(snapInfo *snap.Info, devMode bool, repo *interfaces.Repo
 			logger.Noticef("cannot reload systemd state: %s", err)
 		}
 	}
-	// Start any new services
+	// Start and enable any new services
 	for _, service := range changed {
-		err := systemd.Start(service)
-		if err != nil {
-			logger.Noticef("cannot start service %q: %s", service, err)
+		if err := systemd.EnableNow(service); err != nil {
```

Comment:
Only if the service file _changed!_ This is really the behavior we want I think.